system-contracts/contracts/precompiles/CodeOracle.yul

```diff
@@ -130,6 +130,14 @@ object "CodeOracle" {
 
             let versionedCodeHash := calldataload(0)
 
+            if eq(shr(240, versionedCodeHash), 0x0202) {
+                // 7702 delegation; the bytecode is stored in the hash
+                // It is guaranteed to be 23 bytes long.
+                let bytecode := shl(72, versionedCodeHash)
+                mstore(0, bytecode)
```

Comment:
Note: we don't have actual decommit here. Double check side effects (decommit handling and caching is a bit messy in EraVM)

Reply:
I think it should be ok. Only implication is that this memory page won't be deleted by server